zipkin-ui/js/component_data/default.js

```diff
@@ -21,7 +22,7 @@ export default component(function DefaultData() {
     const query = convertToApiQuery(window.location.search);
     const serviceName = query.serviceName;
     if (serviceName) {
-      const apiURL = `/zipkin/api/v1/traces?${queryString.stringify(query)}`;
+      const apiURL = `${contextRoot}api/v1/traces?${queryString.stringify(query)}`;
```

Comment:
There's a lot of code duplication going on here... maybe we should make a `apiClient.js` which basically just accepts relative paths and wraps `$.ajax`?

Reply:
Seems reasonable. I'll take a look at it.

Reply:
Done. As a matter of fact, the `$.ajax` call with relative paths use the `<base>` tag and as such there was no need to wrap it as you suggested.